en/docs/reference/connectors/cdc-inbound-endpoint/cdc-inbound-endpoint-config.md

```diff
@@ -0,0 +1,182 @@
+  </tr>
+  <tr>
+    <td class="tg-0pky">sequential</td>
+    <td class="tg-0pky">true</td>
+    <td class="tg-0pky">Whether the messages need to be polled and injected sequentially or not. By default this is set to <code>true</code>.</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">connector.name</td>
+    <td class="tg-0pky">-</td>
+    <td class="tg-0pky">Unique name for the connector. If not present, inbound endpoint name is considered as the connector name. </td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">snapshot.mode</td>
+    <td class="tg-0pky">initial</td>
+    <td class="tg-0pky">Specifies the criteria for running a snapshot when the connector starts. Possible settings are: always, initial, initial_only, schema_only, no_data, recovery, when_needed, configuration_based, custom.</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">connector.class</td>
+    <td class="tg-0pky">-</td>
+    <td class="tg-0pky">The name of the Java class for the connector.<br>
+Example: <br>
+For MySQL database, <code>io.debezium.connector.mysql.MySqlConnector</code><br>
+For PostgreSQL database, <code>io.debezium.connector.postgresql.PostgresConnector</code><br>
+For Oracle database, <code>io.debezium.connector.oracle.OracleConnector</code><br>
+For Db2 database, <code>io.debezium.connector.db2.Db2Connector</code></td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">topic.prefix</td>
+    <td class="tg-0pky">-</td>
+    <td class="tg-0pky">Topic prefix that provides a namespace for the database server that you want Debezium to capture. The prefix should be unique across all other connectors, since it is used as the prefix for all Kafka topic names that receive records from this connector. Only alphanumeric characters, hyphens, dots and underscores must be used in the database server logical name.</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">schema.history.internal</td>
+    <td class="tg-0pky">KafkaSchemaHistory</td>
+    <td class="tg-0pky">The name of the Java class that is responsible for persistence of the database schema history.
+It must implement <…>.SchemaHistory interface.
+
+Refer : (https://debezium.io/documentation/reference/stable/development/engine.html) for more information. For database related options check (https://debezium.io/documentation/reference/stable/operations/debezium-server.html#debezium-source-configuration-properties)</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">schema.history.internal.file.filename</td>
+    <td class="tg-0pky">-</td>
+    <td class="tg-0pky">This value is required only if **io.debezium.storage.file.history.FileSchemaHistory** was provided for the <code>schema.history.internal</code> value. You need to specify the path to a file where the database schema history is stored.
```

Comment:
Why using bold for `io.debezium.storage.file.history.FileSchemaHistory`? Best practice is to use bold for UI texts, isn't it @DinithiDiaz ?

Reply:
Yes. @chathuranga-jayanath-99 Please don't make the text bold unless it's UI text.